File: TI1.3 REST_JAVA_CONUNI_GRP3/01 SERVIDOR/src/main/java/ec/edu/espe/conversionapi/enums/VolumeUnit.java
package ec.edu.espe.conversionapi.enums;

/**
 * Unidades de volumen
 */
public enum VolumeUnit {
    LITERS("Litros"),
    MILLILITERS("Mililitros"),
    CUBIC_METERS("Metros Cúbicos"),
    GALLONS("Galones"),
    QUARTS("Cuartos"),
    PINTS("Pintas"),
    CUPS("Tazas");

    private final String displayName;

    VolumeUnit(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: TI1.4 REST_DOTNET_CONUNI_GRP3/02 CLIMOV/convrtfull/app/src/main/java/ec/edu/espe/conv_rtfull/data/model/VolumeUnit.kt
package ec.edu.espe.conv_rtfull.data.model

/**
 * Unidades de volumen
 */
enum class VolumeUnit(val displayName: String, val code: Int) {
    LITERS("Litros", 0),
    MILLILITERS("Mililitros", 1),
    CUBIC_METERS("Metros Cúbicos", 2),
    GALLONS("Galones", 3),
    QUARTS("Cuartos", 4),
    PINTS("Pintas", 5),
    CUPS("Tazas", 6)
}
